events/test.py

```diff
@@ -0,0 +1,118 @@
+import unittest
+import datetime
+import app
+import mongomock
+
+
+example_time = datetime.datetime(2019, 6, 11, 10, 33, 1, 100000)
```

Comment:
Make this CONSTANT_CASE?

Reply:
`pylint -v app.py` should warn you against this.

Reply:
Done in https://github.com/knative-portability/large-events/pull/4/commits/fb026b7885c5f190e191046cdac7c02cbfa72f9d